content/engineering/our-approach/release-checklist.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Release checklist
+sidenav: true
+sticky_sidenav: true
+tags: engineering
+permalink: /engineering/our-approach/release-checklist/
+redirect_from:
+  - engineering/release-checklist/
+layout: layouts/page
+eleventyNavigation:
+  parent: engineering_approach
+  key: Release checklist
+  order: 10
+  title: Release checklist
+subnav:
+  - text: Software launch checklist
+    href: "#software-launch-checklist"
+  - text: What should it look like?
+    href: "#what-should-it-look-like"
+  - text: Questions to consider
+    href: "#questions-to-consider"
+  - text: Resources
+    href: "#resources"
+---
+## Software launch checklist
+
```

Comment:
```suggestion
```

@kewlguy781 Suggest deleting this because it looks like it duplicates the `title` element in the preview build: https://federalist-8bc0b42c-5fd0-4ae8-9366-cd9c265a4446.sites.pages.cloud.gov/preview/18f/guides/jon-releasecheck/engineering/our-approach/release-checklist/

Reply:
@kewlguy781 -- take a look at this suggestion to remove the duplicate title.